src/components/icon/icon.spec.ts

```diff
@@ -251,6 +253,53 @@ describe('Icon broadcast service', () => {
 
       expect(events.length).to.equal(0);
     });
+
+    it('when multiple broadcast services are initialized they should not send sync events to each other.', async () => {
+      const collections = createIconDefaultMap<string, SvgIcon>();
+      const references = createIconDefaultMap<string, IconMeta>();
+
+      const evt1 = [];
+      const broadcast1 = new IconsStateBroadcast(collections, references);
+      (broadcast1 as any).channel.addEventListener(
+        'message',
+        (e: MessageEvent<BroadcastIconsChangeMessage>) => {
+          evt1.push(e);
+        }
+      );
+
+      const evt2 = [];
+      const broadcast2 = new IconsStateBroadcast(collections, references);
+      (broadcast2 as any).channel.addEventListener(
+        'message',
+        (e: MessageEvent<BroadcastIconsChangeMessage>) => {
+          evt2.push(e);
+        }
+      );
+
+      const evt3 = [];
+      const broadcast3 = new IconsStateBroadcast(collections, references);
+      (broadcast3 as any).channel.addEventListener(
+        'message',
+        (e: MessageEvent<BroadcastIconsChangeMessage>) => {
+          evt3.push(e);
+        }
+      );
+
+      // a peer is requesting a state sync
+      channel.postMessage({ actionType: ActionType.SyncState });
+      await nextFrame();
```

Comment:
Also this seems to be flaky depending on the frame timing the messages are not always received. Sadly this:
```suggestion
      await aTimeout(20);
```
seems to be way more stable.